files-community/Files
# Fix: Fixed issue where opening properties would download OneDrive files

**Resolved / Related Issues**
   Closes #13006
Closes #13593


**Validation**
How did you test these changes?
   1. Open properties for a non OneDrive file/folder and confirm "Size on disk" is calculated
   2. Open properties for a downloaded OneDrive file/folder and confirm "Size on disk" is calculated
   3. Open properties for a non downloaded OneDrive file/folder and confirm "Size on disk" isn't calculated
   4. Open hash page for online file and confirm the file isn't downloaded
   5. Open hash page for offline file and confirm the hash is displayed

**Screenshots (optional)**
Add screenshots here.
